tests/testthat/test-average_wait.R

```diff
@@ -0,0 +1,56 @@
+# # Anticipated test from the error handling that Matt Dray is drafting
+# test_that("function catches null values and reports error", {
+#   em <- "target_wait(): no error message when function is run with no inputs."
+#   expect_error(target_wait(), em)
+# })
+#
+# # Anticipated test from the error handling that Matt Dray is drafting
+# test_that("function catches mismatched input lengths", {
+#   em <- "target_wait(): no error message when functions inputs are of different length."
+#   expect_error(target_wait(c(22,25,26), c(4, 3)), em)
+# })
+#
+# # Anticipated test from the error handling that Matt Dray is drafting
+# test_that("it returns an error if either input aren't numeric", {
+#   in1 <- Sys.Date()
+#   in2 <- 4
+#
+#   em <- "target_wait(): all inputs must be numeric."
+#   expect_error(target_wait(in1, in2), em)
+# })
+
+test_that("it returns an expected result with fixed single values, against arithmetic", {
+  target_wait <-
```

Comment:
Maybe I'm misunderstanding, but is there a reason for these hanging assignments?

Reply:
Good spot.  Copy paste errors.  Will remove.